client/src/Components/ContainerApp/ContainerApp.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Paper } from '@material-ui/core';
+
+import ContainerAppStyles from './ContainerAppStyles';
+
+const ContainerApp = props => {

```

Comment:
Nazwa ContainerApp kojarzy mi się bardziej z czymś co owija całą apkę. Może lepiej nazwać to np: MainBox. Do przemyślenia